resources/external_control.urscript

```diff
@@ -160,16 +223,23 @@ def cubicSplineRun(end_q, end_qd, time, is_last_point=False):
     local coefficients4 = [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
     local coefficients5 = [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
     jointSplineRun(coefficients1, coefficients2, coefficients3, coefficients4, coefficients5, time, is_last_point)
+  else:
+    trajectory_result = TRAJECTORY_RESULT_CANCELED
   end
 end
 
 def quinticSplineRun(end_q, end_qd, end_qdd, time, is_last_point=False):
   local str = str_cat(end_q, str_cat(end_qd, str_cat(end_qdd, time)))
   textmsg("Quintic spline arg: ", str)
 
-  # Zero time means zero length and therefore no motion to execute
-  if time > 0.0:
-    local start_q = get_target_joint_positions()
+  local start_q = get_target_joint_positions()
+  # Zero time means infinite velocity to reach the target and is therefore impossible
+  if time <= 0.0:
```

Comment:
I am afraid that some people are putting the robot's current position as the first node in the trajectory.
Maybe we could ignore this not with the time of zero if:
1. it is the first node in the path
1. it is very close to the robot's current position